public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -285,19 +302,32 @@ export class PanelInspector extends PureComponent<Props, State> {
     tabs.push({ label: 'Raw JSON', value: InspectTab.Raw });
 
     return (
-      <Drawer title={panel.title} onClose={this.onDismiss}>
-        <TabsBar>
-          {tabs.map((t, index) => {
-            return (
-              <Tab
-                key={`${t.value}-${index}`}
-                label={t.label}
-                active={t.value === tab}
-                onChangeTab={() => this.onSelectTab(t)}
-              />
-            );
-          })}
-        </TabsBar>
+      <InspectHeader
+        tabs={tabs}
+        tab={tab}
+        stats={stats}
+        onSelectTab={this.onSelectTab}
+        onClose={this.onDismiss}
+        panel={this.props.panel}
+        onToggleExpand={this.onToggleExpand}
+        isExpanded={this.state.drawerWidth === '100%'}
+      />
+    );
+  };
+
+  render() {
+    const { panel } = this.props;
+    const { last, tab, drawerWidth } = this.state;
+    const styles = getStyles();
+
+    const error = last?.error;
+    if (!panel) {
```

Comment:
This seems weird - why to check whether or not panel exists?

Reply:
Yeah, that was weird as the same check exists in componentDitMount